test/factories.rb

```diff
@@ -0,0 +1,23 @@
+# This will guess the User class
+FactoryGirl.define do  factory :stach, :class => 'Stache' do
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.